docs/development/seed_network_policies.md

```diff
@@ -108,8 +108,7 @@ allow-to-dns                      networking.gardener.cloud/to-dns=allowed
 allow-to-loki                     networking.gardener.cloud/to-loki=allowed                       
 allow-to-private-networks         networking.gardener.cloud/to-private-networks=allowed                    
 allow-to-public-networks          networking.gardener.cloud/to-public-networks=allowed                     
-allow-to-runtime-apiserver        networking.gardener.cloud/to-runtime-apiserver=allowed                      
-deny-all                          networking.gardener.cloud/to-all=disallowed                              
```

Comment:
While the `deny-all` network policy was changed, i.e. its pod selector was removed, I still expect it to be present in the `garden` namespace after this change. Therefore, I would also expect it to be in this list, just without a pod selector (or with something like `<none>`).
WDYT?

Reply:
If we kept this `deny-all` in `garden` namespace then we would need to adapt all components and properly specify `NetworkPolicy`s for them. That's definitely the plan, but not as part of this PR. It will follow in the future.

Reply:
Ok, I misread the namespaces the `deny-all` policy is applied to. (There is too much `Garden` in the constants.)

Well, it is unfortunate to remove the deny-all policy, but the way it was implemented in `garden` namespace was rather special. Was any pod annotated at all with this annotation?

Reply:
No, there was no user. From https://github.com/gardener/gardener/pull/7408/commits/d4150f4c960778b70dbd55f35c41d92162067986:

> This effectively drops the `deny-all` policy in the `garden` namespace. It was never used because it was only selecting pods with certain labels (and no pods ever had such labels). In the future, we will deploy it also into the `garden` namespace.